portfolio/src/main/java/com/google/sps/utils/CommentUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+/** Auxiliary functions and variables to support comment feature. */
+public class CommentUtils {
```

Comment:
There's nothing about comment here. Maybe just Servlet util?
Make this class final. And since this class only contains static methods, make it non initializable by creating a private constructor.
public final class ServletUtils {
  private ServletsUtil() {}

  // static methods

}

Reply:
wow new knowledge get! thx! Fixed